data/com.endlessm.DemoMode.rules.in

```diff
@@ -0,0 +1,13 @@
+polkit.addRule(function(action, subject) {
+    if (subject.user !== 'demo-guest')
+        return undefined;
+
+    if ((action.id == "org.freedesktop.Flatpak.app-install" ||
+         action.id == "org.freedesktop.Flatpak.runtime-install"||
+         action.id == "org.freedesktop.Flatpak.app-uninstall" ||
+         action.id == "org.freedesktop.Flatpak.runtime-uninstall") &&
+        subject.active == true && subject.local == true &&
+        (subject.isInGroup("@privileged_group@") || (subject.user == "demo-guest"))) {
+            return polkit.Result.YES;
```

Comment:
We already check if the user is `demo-guest` above. Also, what is `@privileged_group@` defined to?